app/models/picture.rb

```diff
@@ -0,0 +1,31 @@
+class Picture < ActiveRecord::Base
+
+  belongs_to :user
+  has_many :comments, dependent: :destroy
+  has_many :likes, dependent: :destroy
+  validates :title, :image, presence: true
+
+  has_attached_file :image,
+                    :styles => {  :medium => "800x800>",
+                                  :thumb => "200x200>" },
+                    :default_url => "/images/:style/missing.jpeg"
+  validates_attachment_content_type :image, :content_type => /\Aimage\/.*\Z/
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)<br>Style/RegexpLiteral: Use `%r` around regular expression. (https://github.com/bbatsov/ruby-style-guide#percent-r)